plugins/network-policy-audit/tests/validation/analysis_modes_test.sh

```
@@ -0,0 +1,179 @@
+echo "Test ID: ${TEST_ID}"
+echo ""
+
+set +e  # Don't exit on errors
+
+# Start test log
+{
+    echo "=== ANALYSIS MODES TESTS ==="
+    echo "Date: $(date)"
+    echo ""
+
+    cd ${PLUGIN_DIR}
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)